cmd/copy-file-to-pod/main.go

```diff
@@ -109,6 +117,34 @@ func main() {
 	printOutput("successfully copied %s to %s:%s", fileSrc, pod, fileDest)
 }
 
+func podNameFromFlags(f cmdutil.Factory, podName, podSelector, namespace string) (string, error) {
+	cs, err := f.KubernetesClientSet()
+	if err != nil {
+		return "", err
+	}
+
+	var pod string
+	if podName != "" {
+		pod = podName
```

Comment:
Let's just return here
```suggestion
		return podName
```